sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/AdtQuerySelect.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.DigitalTwins.Core.QueryBuilder
+{
+    /// <summary>
+    /// Query object that already contains a select clause.
+    /// </summary>
+    internal class AdtQuerySelect : AdtQuery
+    {
+        /// <summary>
+        /// Adds the FROM clause and its arugment to the query via the Clauses component.
+        /// </summary>
+        /// <param name="collection"> An enum for the two different collections that users can query from (DT instances or relationships). </param>
```

Comment:
don't specify the count of the collection, you don't want the comment to go obsolete when you add more items to the collection.